test_app/Gemfile.minitest-stdlib

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+gem 'minitest', '~> 5.5.0'
```

Comment:
Should this version constraint match the `mutant-minitest` gemspec? It is `'~> 5.5.0`' here will be `'~> 5.3.0` in the gemspec judging by this thread https://github.com/mbj/mutant/pull/445/files#r48111671